Misc/NEWS.d/next/Library/2024-05-11-23-32-29.gh-issue-118961.06YL16.rst

```diff
@@ -0,0 +1,2 @@
+Fixes random.choices undefined behavior when some weights are not finite or
+negative.
```

Comment:
```suggestion
Fixes :func:`random.choices` undefined behavior when some weights are not finite
or negative.
```

Reply:
Resolved because the PR is closed. Thanks!